File: Android/PlayMatch/app/src/main/java/com/dam/daniel/playmatch/models/Message.java
package com.dam.daniel.playmatch.models;

public class Message {

    private int id;
    private int chat;
    private int userSender;
    private int userReceiver;
    private String message;
    private String date;

    public Message(){ }

    public Message(int id, int chat, int userSender, int userReceiver, String message, String date) {
        this.id = id;
        this.chat = chat;
        this.userSender = userSender;
        this.userReceiver = userReceiver;
        this.message = message;
        this.date = date;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getChat() {
        return chat;
    }

    public void setChat(int chat) {
        this.chat = chat;
    }

    public int getUserSender() {
        return userSender;
    }

    public void setUserSender(int userSender) {
        this.userSender = userSender;
    }

    public int getUserReceiver() {
        return userReceiver;
    }

    public void setUserReceiver(int userReceiver) {
        this.userReceiver = userReceiver;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    @Override
    public String toString() {
        return "Message{" +
                "id=" + id +
                ", chat=" + chat +
                ", userSender=" + userSender +
                ", userReceiver=" + userReceiver +
                ", message='" + message + '\'' +
                ", date='" + date + '\'' +
                '}';
    }
}

File: Backend/PlayMatch/src/models/Message.ts
export class Message{

    chat: number;
    userSender: number;
    userReceiver: number;
    message: string;
    date: string;

    constructor(chat: number, userSender: number, userReceiver: number, message: string, date?: string){
        this.chat = chat;
        this.userSender = userSender;
        this.userReceiver = userReceiver;
        this.message = message;
        this.date = date;
    }

    public getChat()
	{
		return this.chat;
	}

	public setChat(chat: number)
	{
		this.chat = chat;
	}

	public getUserSender()
	{
		return this.userSender;
	}

	public setUserSender(userSender: number)
	{
		this.userSender = userSender;
    }
    
    public getUserReceiver()
	{
		return this.userReceiver;
	}

	public setUserReceiver(userReceiver: number)
	{
		this.userReceiver = userReceiver;
	}

	public getMessage()
	{
		return this.message;
	}

	public setMessage(message: string)
	{
		this.message = message;
	}

	public getDate()
	{
		return this.date;
	}

	public setDate(date: string)
	{
		this.date = date;
	}

}